app/search_builders/hyrax/collection_search_builder.rb

```diff
@@ -20,7 +20,7 @@ def discovery_permissions
 
     # @return [String] Solr field name indicating default sort order
     def sort_field
-      "title_si"
+      "title_ssi"
```

Comment:
Including this fix because I believe it's a typo. 


-     title_si: The suffix _si suggests that this field should be treated as a "Short Integer" based on Solr's default dynamic field definitions.~
-     title_ssi: The suffix _ssi suggests that this field is a "String, stored, and indexed".